--- a/llvm/tools/bugpoint-passes/CMakeLists.txt
+++ b/llvm/tools/bugpoint-passes/CMakeLists.txt
@@ -5,9 +5,9 @@
 # There's no reason to export anything from the plugin.
 set(LLVM_EXPORTED_SYMBOL_FILE ${CMAKE_CURRENT_SOURCE_DIR}/bugpoint.exports)
 
-if(WIN32 OR CYGWIN OR ZOS)
+# if(WIN32 OR CYGWIN OR ZOS)
   set(LLVM_LINK_COMPONENTS Core Support)
-endif()
+# endif()
 
 add_llvm_library( BugpointPasses MODULE BUILDTREE_ONLY
   TestPasses.cpp
